mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss

```diff
@@ -314,8 +314,7 @@
 }
 
 /* Gutenberg bug: Close button is not visible in Safari; override it locally */
-.global-header__navigation .is-menu-open,
-.global-header__search .is-menu-open {
+.global-header__navigation .is-menu-open {
 	overflow: visible;
 
 	& > div {
```

Comment:
I'm not sure if the Safari fix is still necessary here, but the `global-header__navigation .is-menu-open { overflow: visible; }` is, so I've just reverted the change where I added `global-header__search`.

Reply:
Should we also change the comment here? Since this seems not related only to Safari anymore.
Not a blocker, but I spent some time figuring out what this really means.